evalml/tests/component_tests/test_minmax_scaler.py

```diff
@@ -0,0 +1,56 @@
+import pandas as pd
+
+from evalml.pipelines.components import MinMaxScaler
+
+
+def test_minmax_scaler_numeric_only(X_y_binary):
+    X, y = X_y_binary
+    scaler = MinMaxScaler()
+    for col in X:
+        assert min(col) < 0.0 or max(col) > 1.0
+
+    scaler.fit(X, y)
+    X_t = scaler.transform(X)
+    for col in X_t.columns:
+        assert min(X_t[col]) >= 0.0
+        assert max(X_t[col]) <= 1.0
+
+    scaler = MinMaxScaler(feature_range=(-1, 3))
+    X_t = scaler.fit_transform(X, y)
+    for col in X_t.columns:
+        assert -1 <= min(X_t[col]) < 0
+        assert 1 < max(X_t[col]) <= 3.1
```

Comment:
Why is the range here 3.1 instead of 3?

Reply:
There's a small rounding error, so the max value in this case is actually 3.00004. Close enough to not matter, but far enough to break the test if left at 3.

Reply:
Wow, that's interesting that the rounding error would pass sklearn's internal tests. 
This seems fine to me then!